metal/v1/api_projects.go

```diff
@@ -405,6 +405,8 @@ Organization owners can transfer their projects to other organizations.
 	@param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
 	@param id UUID of the project to be transferred
 	@return ApiCreateTransferRequestRequest
+
+Deprecated
```

Comment:
I take it we deprecated transfer requests?  Does that mean customers can't transfer projects from one org to another or can they still do that and this just an old approach we don't want to support?